### ISSUE TITLE
Session does not contain antiforgery ID

### ISSUE THREAD

**Issue by kidfilkings** [psignoret/aad-sso-wordpress#217]
Hi,

As I couldn't find any solution that fits my scenario I'm openning this issue hoping you're still in this project.
My users are getting "Session does not contain antiforgery ID" when trying to log in. I'm pretty sure I have all configurations right in the plugin and permission in Azure Ad.
The thing is: I'm using an Wordpress by WPEngine.com.
I didn't make any change to your solution and that error message makes no sense to me.

**Comment by horaciogaspar** [comment]
I've been experiencing the same error message - running Wordpress on my Centos platform.  I'm also looking for a solution to this issue - it's an intermittent error and not sure how to reproduce the issue.

**Comment by kidfilkings** [comment]
HI,

I was able to solve it reading a few tips in other issues like this.
In my case I'm using a plugin to rename the native file wp-login.php. I was using 'WPS Hide Login' plugin in my new site. In an old site I had 'Rename wp-login.php' plugin and it was working, so I installed this plugin and deactivated the other one and everything worked as expected.

So in case you having this same error message and you're using any plugin that affects the Wordpress native login module try uninstall the plugin and see if it's causing the issue.
